vmdb/product/views/SystemService.yaml

```diff
@@ -20,26 +20,35 @@ db: SystemService
 # Columns to fetch from the main table
 cols:
 - name
-- display_name
-- enable_run_levels
 - running
+- load

```

Comment:
@martinpovolny @gmcculloug 

Host is now using only this one view. Is it ok to extend it with what I collect from systemd? Cause I did the same to host fleecing, it prefers systemd over init now.

Also removing 'display name', seems to me it's used only in windows